File: Java/pertemuan 10/atm.java
import java.util.Scanner;

public class atm {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String nama, benar;
        int saldoAwal = 0, inputSaldo, menu, tarikTunai, setorTunai, transfer;

        System.out.println("=====================================================");
        System.out.print("Masukkan Nama Anda: ");
        nama = input.nextLine();

        System.out.print("Apakah Anda Yakin Ingin Melanjutkan Transaksi? (Y/N): ");
        benar = input.nextLine();

        System.out.println("=====================================================");

        if (benar.equalsIgnoreCase("Y")) {
            System.out.println("=====================================================");
            System.out.println("Selamat Datang Di ATM, " + nama);
            System.out.println("Saldo Anda Saat Ini: Rp " + saldoAwal);
            System.out.println("=====================================================");
        } else {
            System.out.println("=====================================================");
            System.out.println("          Terima Kasih Atas Kunjungan Anda");
            System.out.println("=====================================================");
            return;
        }

        System.out.print("Masukkan Saldo : ");
        inputSaldo = input.nextInt();

        System.out.println("=====================================================");
        System.out.println("NAMA ANDA              : " + nama);
        System.out.println("SALDO ANDA SEKARANG    : Rp " + inputSaldo);
        System.out.println("=====================================================");
        System.out.println();

        System.out.println("===================== MENU ATM ======================");
        System.out.println("1. CEK SALDO");
        System.out.println("2. TARIK TUNAI");
        System.out.println("3. SETOR TUNAI");
        System.out.println("4. TRANSFER");
        System.out.println("5. KELUAR");
        System.out.println("=====================================================");

        System.out.print("Pilih Menu : ");
        menu = input.nextInt();

        switch (menu) {
            case 1:
                System.out.println("=====================================================");
                System.out.println("Saldo Anda Saat Ini: Rp " + inputSaldo);
                System.out.println("=====================================================");
                break;

            case 2:
                System.out.println("Saldo Anda Saat Ini: Rp " + inputSaldo);
                System.out.print("Masukkan Jumlah Tarik Tunai: ");
                tarikTunai = input.nextInt();
                System.out.println("=====================================================");
                System.out.println("Saldo Anda Saat Ini: Rp " + (inputSaldo - tarikTunai));
                System.out.println("=====================================================");
                break;

            case 3:
                System.out.println("Saldo Anda Saat Ini: Rp " + inputSaldo);
                System.out.print("Masukkan Jumlah Setor Tunai: ");
                setorTunai = input.nextInt();
                System.out.println("=====================================================");
                System.out.println("Saldo Anda Saat Ini: Rp " + (inputSaldo + setorTunai));
                System.out.println("=====================================================");
                break;

            case 4:
                System.out.println("Saldo Anda Saat Ini: Rp " + inputSaldo);
                System.out.print("Masukkan Jumlah Transfer: ");
                transfer = input.nextInt();
                System.out.println("=====================================================");
                System.out.println("Saldo Anda Saat Ini: Rp " + (inputSaldo - transfer));
                System.out.println("=====================================================");
                break;

            case 5:
                System.out.println("=====================================================");
                System.out.println("          Terima Kasih Atas Kunjungan Anda");
                System.out.println("=====================================================");
                break;

            default:
                System.out.println("=====================================================");
                System.out.println("Menu Tidak Tersedia");
                System.out.println("=====================================================");
                break;
        }
    }
}

File: c++/pertemuan 10/atm.cpp
#include <iostream>
using namespace std;

main() {
    string nama;
    string benar;
    int saldoAwal = 0, inputSaldo, menu, tarikTunai, setorTunai, transfer;

    cout << "=====================================================" << endl;
    cout << "Masukkan Nama Anda: ";
    cin >> nama;
    cout << "Apakah Anda Yakin Ingin Melanjutkan Transaksi? (Y/N): ";
    cin >> benar;
    cout << "=====================================================" << endl;

    if (benar == "Y" || benar == "y") {
        system("cls");
        cout << "=====================================================" << endl;
        cout << "Selamat Datang Di ATM, " << nama << endl;
        cout << "Saldo Anda Saat Ini: Rp " << saldoAwal << endl;
        cout << "=====================================================" << endl;
    } else {
        system("cls");
        cout << "=====================================================" << endl;
        cout << "          Terima Kasih Atas Kunjungan Anda           " << endl;
        cout << "=====================================================" << endl;
        return 0;
    }


    cout << "Masukkan Saldo : ";
    cin >> inputSaldo;
    cout << "=====================================================" << endl;
    cout << "NAMA ANDA              : " << nama << endl;
    cout << "SALDO ANDA SEKARANG    : Rp " << inputSaldo << endl;
    cout << "=====================================================" << endl;
    cout << endl;
    cout << "===================== MENU ATM ======================" << endl;
    cout << "1. CEK SALDO" << endl;
    cout << "2. TARIK TUNAI" << endl;
    cout << "3. SETOR TUNAI" << endl;
    cout << "4. TRANSFER" << endl;
    cout << "5. KELUAR" << endl;
    cout << "=====================================================" << endl;
    cout << "Pilih Menu : ";
    cin >> menu;
    
    switch (menu){
        case 1:
        system("cls");
        cout << "=====================================================" << endl;
        cout << "Saldo Anda Saat Ini: Rp " << inputSaldo << endl;
        cout << "=====================================================" << endl;
        break;

        case 2:
        system("cls");
        cout << "=====================================================" << endl;
        cout << "Saldo Anda Saat Ini: Rp " << inputSaldo << endl;
        cout << "Masukkan Jumlah Tarik Tunai: ";
        cin >> tarikTunai;
        cout << "=====================================================" << endl;
        cout << "Saldo Anda Saat Ini: Rp " << inputSaldo - tarikTunai << endl;
        cout << "=====================================================" << endl;
        break;

        case 3:
        system("cls");
        cout << "=====================================================" << endl;
        cout << "Saldo Anda Saat Ini: Rp " << inputSaldo << endl;
        cout << "Masukkan Jumlah Setor Tunai: ";
        cin >> setorTunai;
        cout << "=====================================================" << endl;
        cout << "Saldo Anda Saat Ini: Rp " << inputSaldo + setorTunai << endl;
        cout << "=====================================================" << endl;
        break;

        case 4:
        system("cls");
        cout << "=====================================================" << endl;
        cout << "Saldo Anda Saat Ini: Rp " << inputSaldo << endl;
        cout << "Masukkan Jumlah Transfer: ";
        cin >> transfer;
        cout << "=====================================================" << endl;
        cout << "Saldo Anda Saat Ini: Rp " << inputSaldo - transfer << endl;
        cout << "=====================================================" << endl;
        break;
        
        case 5:
        system("cls");
        cout << "=====================================================" << endl;
        cout << "          Terima Kasih Atas Kunjungan Anda           " << endl;
        cout << "=====================================================" << endl;
        break;
        
    default:
        system("cls");
        cout << "=====================================================" << endl;
        cout << "Menu Tidak Tersedia" << endl;
        cout << "=====================================================" << endl;
        break;
    }

    return 0;
}